.gitignore

```diff
@@ -5,4 +5,8 @@
 # The packages folder can be ignored because of Package Restore
 **/packages/*
 bin/
-obj/
\ No newline at end of file
+obj/
+.DS_Store
+Notify.userprefs
+NotifyTests/.DS_Store
+environment.sh
```

Comment:
Could group these ignored files and create headings like in other Notify repos

Reply:
Done